db/schema.rb

```diff
@@ -37,6 +37,14 @@
     t.index ["user_id", "user_type"], name: "user_index"
   end
 
+  create_table "consistency_checks", force: :cascade do |t|
+    t.integer "trainee_id"
+    t.datetime "contact_last_updated_at"
+    t.datetime "placement_assignment_last_updated_at"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
```

Comment:
cross pollination...

Reply:
whoops!!! Sorry, Ill get rid of this now 👍